### PR TITLE
Added Image Gallery Listing Metadata (#127).

### PR DESCRIPTION
**Pull request by robgietema**

### CODE REVIEW

File: src/icons/fullscreen.svg
Comment (ichim-david) [inline src/icons/fullscreen.svg:1]: @robgietema @sneridagh  I think that volto-light-theme should introduce a standard for the icon size and keep it consistent across each icon making it easier to size when passing in a width value.

If you look at a dedicated icon library such as remix https://remixicon.com/ each icon has a space of 24x24 to place an icon inside that area.
Pastanaga Icons for instance are all 36x36px.

In this pull request fullscreen.svg and play.svg are 50x50px while left-key right-key are 39x60.

Whichever standard of sizes you want to adopt please keep it consistent and add some info about it

Comment (tisto) [reply src/icons/fullscreen.svg:1]: @ichim-david good point! Let's adopt the 36x36px standard of Pastanaga Icons then.

Comment (robgietema) [reply src/icons/fullscreen.svg:1]: Agree, all icons are now 36x36px.

**Review by sneridagh — APPROVED** [review]
LGTM

File: src/components/Blocks/Listing/ImageGallery.jsx [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+      settings.imageObjects.includes(content['@type']) && content.image_field,
+  );
+  const imagesInfo = renderItems.map((item) => {
+    return {
+      original: `${flattenToAppURL(item['@id'])}/@@images/${
+        item.image_field
+      }/large`,
+      thumbnail: `${flattenToAppURL(item['@id'])}/@@images/${
+        item.image_field
+      }/thumb`,
```
Comment (davisagli) [inline src/components/Blocks/Listing/ImageGallery.jsx:87]: @robgietema @sneridagh We should not be adding new code that uses the shortcut image scale URLs.

**Pull request merged**